codec-redis/src/main/java/io/netty/handler/codec/redis/RedisMessagePool.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.netty.handler.codec.redis;
+
+import io.netty.buffer.ByteBuf;
+
+/**
+ * A strategy interface for caching {@link RedisMessage}s.
+ */
+public interface RedisMessagePool {
```

Comment:
Right.. I want to prevent users confusing because `Redis` is cache.. I know `Pool` is not a good word too. But, I think it's (little) better than `Cache`. If you have a keyword for this, let me know.